src/Validation.hs-boot

```diff
@@ -0,0 +1,13 @@
+module Validation
+    ( Validation (..)
+    , validation
+    ) where
+
+
+data Validation e a
+    = Failure e
+    | Success a
+
+instance (Semigroup e) => Applicative (Validation e)
```

Comment:
This is very cool! I never used this Haskell feature before ✨

Reply:
Me too! very excited :heart_eyes: